rita_common/src/blockchain_oracle/mod.rs

```diff
@@ -326,7 +334,7 @@ pub fn low_balance() -> bool {
     let balance = payment_settings.balance.clone();
     let balance_warning_level = payment_settings.balance_warning_level;
 
-    balance < balance_warning_level
+    balance.unwrap_or_else(|| 0u64.into()) < balance_warning_level
```

Comment:
if the balnce is none this function should retun false and not do the comparison. Becuase we don't know if we have a low balance.